ports/mimxrt/boards/MIMXRT1020_EVK/mpconfigboard.mk

```diff
@@ -2,6 +2,25 @@ MCU_SERIES = MIMXRT1021
 MCU_VARIANT = MIMXRT1021DAG5A
 
 JLINK_PATH ?= /media/RT1020-EVK/
+JLINK_COMMANDER_SCRIPT = $(BUILD)/script.jlink
+
+
+ifdef JLINK_IP
+JLINK_CONNECTION_SETTINGS = -IP $(JLINK_IP)
+else
+JLINK_CONNECTION_SETTINGS = 
+endif
+
+
+deploy_jlink: $(BUILD)/firmware.hex
+	$(Q)$(TOUCH) $(JLINK_COMMANDER_SCRIPT)
```

Comment:
is this touch command needed if the script is anyway rebuild from scratch below?

Reply:
Most likely not. I will take a look and remove if I can.

Reply:
removed